Cargo.toml

```diff
@@ -28,6 +28,10 @@ path = "src/examples/ssd1306-oled.rs"
 name = "oled-bus"
 path = "src/examples/ssd1306-oled-bus.rs"
 
+[[bin]]
```

Comment:
Adding a new binary to the project - it builds this as the root main and pulls in dependencies as necessary.  Binaries don't bloat each other unless they actually share dependencies.